cleanlab/classification.py

```diff
@@ -623,26 +683,33 @@ def find_label_issues(
 
         if self.verbose:
             print("Using predicted probabilities to identify label issues ...")
-        # Get the boolean mask of the label issues
-        self.label_issues_mask = filter.find_label_issues(
+        label_issues_mask = filter.find_label_issues(
             labels,
             pred_probs,
             **self.find_label_issues_kwargs,
         )
+        label_quality_scores = get_label_quality_scores(
+            labels, pred_probs, **self.label_quality_scores_kwargs
+        )
+        self.label_issues_df = pd.DataFrame(
+            {"is_label_issue": label_issues_mask, "label_quality": label_quality_scores}
+        )
+        if pred_probs is not None:
+            self.label_issues_df["predicted_class"] = pred_probs.argmax(axis=1)
         if self.verbose:
-            print(f"Identified {np.sum(self.label_issues_mask)} examples with label issues.")
+            print(f"Identified {np.sum(label_issues_mask)} examples with label issues.")
 
-        return self.label_issues_mask
+        return self.label_issues_df
 
     def get_label_issues(self):
-        """Accessor. Returns `label_issues_mask` if previously already computed."""
+        """Accessor. Returns `label_issues_df` if previously already computed."""
```

Comment:
Add Returns pd.DataFrame (short description) to the docstring.

just use this as the accessor i mention above (`get_label_issues_df`)... the current name `get_label_issues` is fine

Reply:
added